worlds/__init__.py

```diff
@@ -1,13 +1,17 @@
+import dataclasses
 import importlib
 import os
+import pkgutil
 import sys
+import time
 import warnings
 import zipimport
-import time
-import dataclasses
-from typing import Dict, List, TypedDict, Optional
+from typing import Dict, Iterable, List, Optional, Tuple, TypedDict, Union
```

Comment:
```suggestion
from typing import Container, Dict, List, Optional, Tuple, TypedDict, Union
```
(see associated change below)